pkg/apiserver/statement/queries.go

```diff
@@ -17,15 +17,93 @@ import (
 	"fmt"
 	"regexp"
 	"sort"
+	"strconv"
 	"strings"
 
 	"github.com/jinzhu/gorm"
 )
 
 const (
-	statementsTable = "INFORMATION_SCHEMA.CLUSTER_STATEMENTS_SUMMARY_HISTORY"
+	statementsTable        = "INFORMATION_SCHEMA.CLUSTER_STATEMENTS_SUMMARY_HISTORY"
+	stmtEnableVar          = "tidb_enable_stmt_summary"
+	stmtRefreshIntervalVar = "tidb_stmt_summary_refresh_interval"
+	stmtHistroySizeVar     = "tidb_stmt_summary_history_size"
 )
 
+type sqlVariable struct {
+	Name  string `gorm:"column:Variable_name"`
+	Value string `gorm:"column:Value"`
+}
+
+// How to get sql variables by GORM
+// https://github.com/jinzhu/gorm/issues/2616
+func querySQLIntVariable(db *gorm.DB, name string) (int, error) {
+	var variables []sqlVariable
+	err := db.Raw("SHOW GLOBAL VARIABLES LIKE ?", name).Scan(&variables).Error
```

Comment:
better to use:

```go
db.Raw(`SELECT @@GLOBAL.tidb_enable_stmt_summary AS value`).Pluck("value", &....)
```

https://gorm.io/docs/query.html#Pluck

Reply:
Oh, good to know that we can use `SELECT @@GLOBAL.tidb_enable_stmt_summary ...` grammar to get the variable, I will try it. Have tried to use `Pluck` and `SHOW GLOBAL ...` together but failed before.

Reply:
done!